feat: 메뉴 전체 조회 API

re-mora opened the pull request:
## #️⃣ 해당 변경 사항과 연관된 이슈는 무엇인가요?

> #64 

## ✏️ 작업한 내용을 간략히 설명해 주세요!
메뉴 전체 조회 api 생성 
메뉴 전체 조회 테스트 

## 🛠️ 변경을 위해 진행한 작업 내용에는 어떤 것이 있나요?

- 📌 Controller ,service ,repo 메뉴 전체 조회 관련된 코드 추가 
- 📌 http-client 를 통한 메뉴 전체 조회 테스트 코드 추가 
- 📌 

## 📝 변경 사항을 확인하기 위해 테스트한 방법을 설명해 주세요.

> 어떤 방법으로 테스트했는지 알려주세요.

## ✅ 해당 기능에 대한 테스트가 작성되었나요?

- [ ] 🟢 작성
- [x ] 🟡 일부 구현
- [ ] 🔴 미 작성

## 💬 리뷰 요구사항

> 같이 논의했으면 하는 사항이 있으신가요?

## 📚 참고 자료

> 구현에 참고한 자료가 있다면 공유해주세요!

---

## 종료할 이슈는 무엇인가요?

close #64

re-mora committed:
fix(#64): sync to find-all-menu tasks

re-mora committed:
fix(#64): sync complete

re-mora committed:
fix(#64): rollback last commit changes

eggnee commented on src/main/java/com/sparta/baedallegend/domains/menu/controller/MenuController.java:
```diff
@@ -25,4 +33,15 @@ public ResponseEntity<Void> createMenu(
 
+	@GetMapping("/menu/{shopId}")
+	public Page<FindAllMenuResponse> findAllMenu(
+		@PathVariable UUID shopId,
```
String 으로 받아와서 변환하는 게 좋을 것 같습니다!

uzjaee replied:
반영한 뒤 말씀드리고 merge 하겠습니다!

uzjaee replied:
String 으로 변환하였으며 변환후 정상작동 테스트 까지 완료했습니다.

eggnee reviewed (changes requested):
고생하셨습니다! comment 남긴 부분만 수정하시면 될 것 같아요

uzjaee committed:
chore(#64) 메뉴 전체 조회 api
shopID type : UUID > String
서비스 내 UUID.fromString()을 통해 변환
http client 테스트 코드 통합 ( 메뉴생성 메뉴 조회)

re-mora reviewed (commented):
고생하셨습니다! 👍 

제가 올린 PR 이라 approve가 안되네요..

혹시 추가적인 사항이 없다면 머지 하셔도 될 것 같아요!
